docs/README.md

```diff
@@ -1,85 +1,95 @@
+We built Lockbook for everyday use because we use Lockbook every day. We need a note-taking app that doesn't make trade-offs concerning speed, stability, efficiency, device integration, or delightfulness. The only way to have that is to put in the effort, including writing native apps for every platform. We can't wait for you to try it.
 
 ### Secure
-Keep your thoughts to yourself. Your notes are encrypted with keys that are generated on your devices and stay on your devices. The only people that can see your notes are you and the users you share them with. No one else, including infrastructure providers, state actors, and Lockbook employees, can see your notes.
+Keep your thoughts to yourself. Your notes are encrypted with keys that are generated on your devices and stay on your devices. The people that can see your notes are you and the users you share them with. The people that can't are everyone else, including infrastructure providers, state actors, and Lockbook employees.
 
 ### Private
-Know your customer? We sure don't. We don't collect your email, phone number, or name. We don't even need a password. Lockbook is for people with better things to worry about than privacy.
+Know your customer? We sure don't. We don't collect your email, phone number, or name. We don't need a password. Lockbook is for people with better things to worry about than privacy.
 
 ### Honest
-Be the customer, not the product. We make money by selling a note-taking app, not your data.
+Be the customer, not the product. We sell a note-taking app, not your data.
 
 | Payment Option | Monthly Fee    |
 |----------------|----------------|
-| Monthly        | $2.99 per 30gb |
+| Monthly        | $2.99 per 30GB |
 
 ### Developer Friendly
-We also provide a CLI tool that will fit right into your favorite chain of piped-together unix commands. Search your notes with `fzf`, edit them with `vim`, and schedule backups with `cron`. When scripting doesn't cut it, use our Rust library for a robust programmatic interface with Lockbook.
+The Lockbook CLI will fit right into your favorite chain of piped-together Unix commands. Search your notes with `fzf`, edit them with `vim`, and schedule backups with `cron`. When scripting doesn't cut it, use our Rust library for a robust programmatic interface.
+
+## How To Install
+### Mobile
+iOS/iPadOS:
+- [App Store](https://apps.apple.com/us/app/lockbook/id1526775001)
+- [Build From Source](./guides/build/apple.md)
+
+Android:
+- [Play Store](https://play.google.com/store/apps/details?id=app.lockbook)
+- [Github Releases](https://github.com/lockbook/lockbook/releases)
+- [Build From Source](./guides/build/android.md)
+
+### Desktop
+macOS:
+- [App Store](https://apps.apple.com/us/app/lockbook/id1526775001)
+- [Github Releases](https://github.com/lockbook/lockbook/releases)
+- [Build From Source](./guides/build/apple.md)
+
+Linux:
+- `yay` (Arch): `yay -S lockbook-desktop`
```

Comment:
`yay` is one of the many ways to install from the _Arch User Repository_. I think it's good and fine to have `yay -S lockbook-desktop` in here, but it should be something like:

- [AUR (Arch)](https://aur.archlinux.org/packages/lockbook-desktop): `yay -S lockbook-desktop`

in terms of priority of info. Similarly below.

Reply:
I formatted aur and snap for desktop and cli using links to their store pages like you did - not sure if there's a similar option for brew